src/main/java/com/fdmgroup/backend_streamhub/livechat/configuration/KafkaProducerConfiguration.java

```diff
@@ -0,0 +1,37 @@
+package com.fdmgroup.backend_streamhub.livechat.configuration;
+
+import org.apache.kafka.clients.producer.ProducerConfig;
+import org.apache.kafka.common.serialization.StringSerializer;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.kafka.core.DefaultKafkaProducerFactory;
+import org.springframework.kafka.core.KafkaTemplate;
+import org.springframework.kafka.core.ProducerFactory;
+import org.springframework.kafka.support.serializer.JsonSerializer;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@Configuration
+public class KafkaProducerConfiguration {
+
+    // set up the configurations for the kafka producer
+    @Bean
+    public Map<String, Object> producerConfigurations() {
+        Map<String, Object> configurations = new HashMap<>();
+        configurations.put(ProducerConfig.BOOTSTRAP_SERVERS_CONFIG, "localhost:9092");
```

Comment:
Consider moving constants to another class as a static final variable so that it can be easily accessed at a later date

Reply:
done